var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -3,10 +3,15 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import pathlib
+import sys
+
 from spack.package import *
 
+IS_WINDOWS = sys.platform == "win32"
```

Comment:
@johnwparent this looks vestigial, like maybe you were going to use it to define the variant, but it didn't end up getting used. Other than that this looks good.